VideoApi/VideoApi/Controllers/ConferenceController.cs

```diff
@@ -94,15 +72,13 @@ public async Task<IActionResult> BookNewConferenceAsync(BookNewConferenceRequest
                 participant.DisplayName = participant.DisplayName.Trim();
             }
 
-            var audioIngestUrl = _featureToggles.HrsIntegrationEnabled() ? 
-                _audioPlatformService.GetAudioIngestUrl(request.CaseTypeServiceId, request.CaseNumber, request.HearingRefId.ToString()) 
-                : _audioPlatformService.GetAudioIngestUrl(request.HearingRefId.ToString());
+            var audioIngestUrl = audioPlatformService.GetAudioIngestUrl(request.CaseTypeServiceId, request.CaseNumber, request.HearingRefId.ToString());
```

Comment:
Can you please also remove the 'non-hrs' overload of the GetAudioIngestUrl function, in the audio platform service

Reply:
done